packages/story-editor/src/app/rightClickMenu/useLayerSelect.js

```diff
@@ -29,12 +29,12 @@ import { Icons } from '@googleforcreators/design-system';
 import { trackEvent } from '@googleforcreators/tracking';
 import styled from 'styled-components';
 import { getDefinitionForType } from '@googleforcreators/elements';
+import { useUnits } from '@googleforcreators/units';
 
 /**
  * Internal dependencies
  */
 import useStory from '../story/useStory';
-import createPolygon from '../canvas/utils/createPolygon';
```

Comment:
So I did not use this function, as it cannot correctly calculate the box because it doesn't have all the information.

We could convert this to a hook, `useElementPolygon`, so it can access the units provider and get the relevant information from there and then use this same hook for the other part of #10519 (for the is-element-covering-entire-canvas calculation), but I wasn't sure if that was the right way to go.

@merapi, you worked on that second part. What do you think is the best way to port this work to that feature?